src/scripts/ci/start_tpm2_simulator.sh

```diff
@@ -107,4 +114,5 @@ if [ -n "$GITHUB_ACTIONS" ]; then
     echo "BOTAN_TPM2_TCTI_CONF=$tcti_conf"                                 >> $GITHUB_ENV
     echo "BOTAN_TPM2_PERSISTENT_KEY_AUTH_VALUE=$test_pwd"                  >> $GITHUB_ENV
     echo "BOTAN_TPM2_PERSISTENT_RSA_KEY_HANDLE=$persistent_rsa_key_handle" >> $GITHUB_ENV
+    echo "BOTAN_TPM2_PERSISTENT_RSA_KEY_HANDLE=$persistent_ecc_key_handle" >> $GITHUB_ENV
```

Comment:
This has to also be forwarded into the test execution. See `ci_build.py` line 462.